plugins/obs-qsv11/common_utils_linux.cpp

```diff
@@ -0,0 +1,274 @@
+#include "common_utils.h"
+
+#include <time.h>
+#include <cpuid.h>
+#include <util/c99defs.h>
+#include <util/dstr.h>
+#include <va/va_drm.h>
+#include <va/va_str.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <string.h>
+#include <dirent.h>
+
+mfxStatus simple_alloc(mfxHDL pthis, mfxFrameAllocRequest *request,
+		       mfxFrameAllocResponse *response)
```

Comment:
I think that project preference is to only use `UNUSED_PARAMETER` if there is no other alternative. That said, I don't know if anyone thinks it's worth blocking on this.

Reply:
See, for example, #8167.

Reply:
Right sorry, my main reservation is that this makes implementing these a bit noiser. E.g. see the implementations in https://github.com/obsproject/obs-studio/commit/4867e00f94063214057e236f0b18232454a2a457

These are not unused because they will never be unused, just that this functionality is not implemented yet (in this PR). It is already in the pipe though.

Reply:
Sure, that's a valid reason to keep them as-is for now. Thanks for clarifying.